### ISSUE TITLE
Move configuration to key-value

### ISSUE THREAD

**Issue by ricardoekm** [ricardoekm/Kyrix#8]
Related to #3 .

Rely on line position for configuration makes it hard to read and write the file and is fragile. This also makes impossible to add comments to the config.

The recommended approach usually is to use a key-value implementation (.json for Javascript or .properties for Java).  Javascript has native support for .json, which makes it the natural choice.

https://github.com/ricardoekm/Kyrix/blob/0ce6879a307f1d16440ef79fa978d9dc529020a2/compiler/src/index.js#L23-L30

Splitting the configuration of the compiler and the back-end and using .json + [Config Module](http://lorenwest.github.io/node-config/) seems a good option.

**Comment by tracyhenry** [comment]
Yes, also tracked here: https://github.com/tracyhenry/Kyrix/issues/64
